core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -79,6 +83,17 @@ private MobileCore(Context context, Options options)
         } else {
             this.httpLayer = options.httpServiceModule;
         }
+
+        sendDefaultMetrics();
+    }
+
+    private void sendDefaultMetrics() {
+        try {
+            MetricsService metrics = getInstance(MetricsService.class);
+            metrics.sendDefaultMetrics();
+        } catch (ConfigurationNotFoundException e) {
```

Comment:
@danielpassos I don't quite like this, but at the moment there is no way to check the configuration for a service exists without having an instance of this service (because we use the `type` method of the instance to get the name of the configuration). What do you think?